certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+                    {
+                      "name": "Authorization",
+                      "value": "Bearer token_value"
+                    },
+                    {
+                      "name": "custom_header",
+                      "value": "header_value"
+                    }
+                  ],
+                  "httpVerb": "POST",
+                  "url": "https://example.com"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "If the item is not valid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "If not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "summary": "Webhook Trigger",
+        "description": "When a operation is performed on an entity in PPM Express",
+        "operationId": "Webhook_Trigger",
+        "x-ms-trigger": "single",
+        "x-ms-trigger-hint": "To see it work, perform any specified action in PPM Express",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "eventTypes": {
+                  "x-ms-summary": "Event Types",
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "enum": [
+                      "Task:Created",
+                      "Task:Updated",
+                      "Task:Deleted",
+                      "Idea:Created",
+                      "Idea:Updated",
+                      "Idea:Deleted",
+                      "Project:Created",
+                      "Project:Updated",
+                      "Project:Deleted",
+                      "KeyDate:Created",
+                      "KeyDate:Updated",
+                      "KeyDate:Deleted"
+                    ],
+                    "title": "",
+                    "x-ms-visibility": "important"
+                  },
+                  "description": "eventTypes"
+                },
+                "headers": {
+                  "x-ms-summary": "Headers",
+                  "x-ms-visibility": "advanced",
+                  "type": "array",
+                  "items": {
+                    "description": "add headers",
+                    "title": "headers",
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "type": "string",
+                        "description": "headers name",
+                        "title": "name",
+                        "default": "custom_header"
+                      },
+                      "value": {
+                        "type": "string",
+                        "description": "headers value",

```

Comment:
@vsorokin-ppmexpress please make sure everything is grammatically correct and is free of typographical errors.
You can refer to [quality English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) and to [connector string guidance](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance).